Libraries/StableDiffusion/Load.swift

```diff
@@ -0,0 +1,459 @@
+    public static let presetSDXLTurbo = StableDiffusionConfiguration(
+        id: "stabilityai/sdxl-turbo",
+        files: [
+            .unetConfig: "unet/config.json",
+            .unetWeights: "unet/diffusion_pytorch_model.safetensors",
+            .textEncoderConfig: "text_encoder/config.json",
+            .textEncoderWeights: "text_encoder/model.safetensors",
+            .textEncoderConfig2: "text_encoder_2/config.json",
+            .textEncoderWeights2: "text_encoder_2/model.safetensors",
+            .vaeConfig: "vae/config.json",
+            .vaeWeights: "vae/diffusion_pytorch_model.safetensors",
+            .diffusionConfig: "scheduler/scheduler_config.json",
+            .tokenizerVocabulary: "tokenizer/vocab.json",
+            .tokenizerMerges: "tokenizer/merges.txt",
+            .tokenizerVocabulary2: "tokenizer_2/vocab.json",
+            .tokenizerMerges2: "tokenizer_2/merges.txt",
+        ],
+        defaultParameters: { EvaluateParameters(cfgWeight: 0, steps: 2) },
+        factory: { hub, sdConfiguration, loadConfiguration in
+            let sd = try StableDiffusionXL(
+                hub: hub, configuration: sdConfiguration, dType: loadConfiguration.dType)
+            if loadConfiguration.quantize {
+                quantize(model: sd.textEncoder, filter: { k, m in m is Linear })
+                quantize(model: sd.textEncoder2, filter: { k, m in m is Linear })
+                quantize(model: sd.unet, groupSize: 32, bits: 8)
+            }
+            return sd
+        }
+    )
+
+    /// See https://huggingface.co/stabilityai/stable-diffusion-2-1-base for the model details and license
+    public static let presetStableDiffusion21Base = StableDiffusionConfiguration(
+        id: "stabilityai/stable-diffusion-2-1-base",
+        files: [
+            .unetConfig: "unet/config.json",
+            .unetWeights: "unet/diffusion_pytorch_model.safetensors",
+            .textEncoderConfig: "text_encoder/config.json",
+            .textEncoderWeights: "text_encoder/model.safetensors",
+            .vaeConfig: "vae/config.json",
+            .vaeWeights: "vae/diffusion_pytorch_model.safetensors",
+            .diffusionConfig: "scheduler/scheduler_config.json",
+            .tokenizerVocabulary: "tokenizer/vocab.json",
+            .tokenizerMerges: "tokenizer/merges.txt",
+        ],
+        defaultParameters: { EvaluateParameters(cfgWeight: 7.5, steps: 50) },
+        factory: { hub, sdConfiguration, loadConfiguration in
+            let sd = try StableDiffusionBase(
+                hub: hub, configuration: sdConfiguration, dType: loadConfiguration.dType)
+            if loadConfiguration.quantize {
+                quantize(model: sd.textEncoder, filter: { k, m in m is Linear })
+                quantize(model: sd.unet, groupSize: 32, bits: 8)
+            }
+            return sd
+        }
+    )
+
+}
+
+// MARK: - Key Mapping
```

Comment:
The weights don't match the MLX model exactly -- this will rename and remove weights as needed (matching the python implementation)